### PR TITLE
doc/rbd: refine "Create a Block Device Pool"

### PR DESCRIPTION
**Pull request by zdover23**
Refine the "Create a Block Device Pool" section of doc/rbd/rados-rbd-cmds.rst.

https://tracker.ceph.com/issues/57001

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by anthonyeleven — APPROVED** [review]

File: doc/rbd/rados-rbd-cmds.rst [hunk 1]
```diff
@@ -18,12 +18,16 @@ Create a Block Device Pool
 
 #. On the admin node, use the ``ceph`` tool to `create a pool`_.
 
-#. On the admin node, use the ``rbd`` tool to initialize the pool for use by RBD::
+#. On the admin node, use the ``rbd`` tool to initialize the pool for use by
+   RBD:
 
-        rbd pool init <pool-name>
+   .. prompt:: bash #
```
[the remark this answers is not in the document]
Comment (zdover23) [reply doc/rbd/rados-rbd-cmds.rst]: Thanks for that, Ilya. I've changed this to the $ prompt.

**Review by anthonyeleven — APPROVED** [review]
Non-blocking suggestion:

`#. On the admin node / toolbox, use`

**Comment by idryomov** [comment]
> Non-blocking suggestion:
> 
> `#. On the admin node / toolbox, use`

I'd drop this bit altogether.  "toolbox" isn't mentioned anywhere in the Ceph documentation at all and "admin node" is only mentioned in a couple of places, inconsistently.  In reality it could be pretty much any node or container.

Just as an example, "admin node" isn't mentioned in the below "Create a Block Device User" section.  However all steps/commands on this page could be executed from the same place -- `rbd pool init` isn't any different from `ceph auth get-or-create` or `rbd create`.

**Comment by zdover23** [comment]
> > Non-blocking suggestion:
> > `#. On the admin node / toolbox, use`
> 
> I'd drop this bit altogether. "toolbox" isn't mentioned anywhere in the Ceph documentation at all and "admin node" is only mentioned in a couple of places, inconsistently. In reality it could be pretty much any node or container.
> 
> Just as an example, "admin node" isn't mentioned in the below "Create a Block Device User" section. However all steps/commands on this page could be executed from the same place -- `rbd pool init` isn't any different from `ceph auth get-or-create` or `rbd create`.

@idryomov: I've removed references to "admin node". Check out the text as it stands now and let me know if it looks right to you.

**Comment by anthonyeleven** [comment]
Fair enough.  The `toolbox` is the Rook equivalent.  We might say something like "a node with the Ceph admin key" but really that's the case for hundreds of commands throughout the docs.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/49307 - Quincy backport
https://github.com/ceph/ceph/pull/49308 - Pacific backport

**Pull request merged**